CONTRIBUTING.md

```diff
@@ -0,0 +1,291 @@
+- Start the description / footer of a commit with `BREAKING-CHANGE:` to trigger a major (x.0.0) version. You also have to add either `fix:` or `feat:` to the main body of the commit when using `BREAKING-CHANGE:`. Do this when the changes directly effect the built files / components used by the end user.
+  See the examples further below to learn how to use the correct syntax.
+
+#### Scope
+
+To make commit messages and the changelog more specific and readable, you have the option to scope your commits by adding a keyword in parentheses that indicates the area or aspect you are working on. This practice helps provide clearer context and organization to the commit history.
+
+##### Examples:
+
+- Adding a new component: `feat(Button): added a new Button component`.
+- Adding documentation for icons: `docs(icons): added new documentation for the icons package`.
+
+#### When to use what keywords
+
+It is crucial to understand the distinctions between the two sections mentioned below. If you wish for commit messages to be included in the changelog, please use `fix:`, `feat:`, or `BREAKING-CHANGE:` as keywords. These keywords indicate changes that impact the users of our NPM packages and are therefore significant to highlight. For any other types of changes that do not directly affect the end user, please utilize a different keyword. If you are uncertain about which keyword to use and the changes are non-user-facing, you can use `chore:` as a default keyword.
+
+##### Added to changelog
+
+- `fix:` Patches a bug in the codebase. Nothing new is introduced in terms of functionality.
+- `feat:` Introduces a new feature to the codebase. A new component is an often use case.
+- `BREAKING-CHANGE:` Introduces a breaking change to existing functionality.
+  - **Examples:**
+    - A component is removed from a package
+    - Functionality of a component is changed in a way that requires the end user to perform an action
+
+##### Not added to changelog
+
+- `build:` Changes that affect the build system or external dependencies (example scopes: rollup, stylelint, npm)
+- `chore:` Other changes that don't modify src or test files
+- `docs:` Documentation only changes
+- `style:` Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
+- `test:` Adding missing tests or correcting existing tests
+- `refactor:` A code change that neither fixes a bug nor adds a feature
+- `revert:` Reverts a previous commit
+- `perf:` A code change that improves performance
+
+---
+
+### How to write and structure your code
+
+To ensure a consistent and enjoyable coding experience for everyone, we have established guidelines for writing our code.
+
+#### Styling with CSS Modules
+
+We use CSS modules to style our components. This prevents naming conflicts by adding a unique prefix to all components.
+A CSS module file is created by adding `.module.css` to the end of the CSS file.
+
+#### Use of design tokens
```

Comment:
Provide a link to the tokens: https://www.designsystemet.no/grunnleggende/designelementer/design-tokens